sb.web.app/src/serverFetch.js

```diff
@@ -1011,15 +1015,17 @@ export async function setStoreData () {
       store.commit('replaceAllMembers', data.allMembers)
     }
 
-    if (data.myCart) {
-      store.commit('replaceMyCart', data.myCart)
+    // removed this because all cart store data's are handled by the setCartData function
 
-      let cartSize = 0
-      for (const item of data.myCart) {
-        cartSize += item.quantity
-      }
-      store.commit('replaceMyCartSize', cartSize)
-    }
+    // if (data.myCart) {
+    //   store.commit('replaceMyCart', data.myCart)
+
+    //   let cartSize = 0
+    //   for (const item of data.myCart) {
+    //     cartSize += item.quantity
+    //   }
+    //   store.commit('replaceMyCartSize', cartSize)
+    // }
```

Comment:
i'd prefer for this to be removed rather than just commented out but not enough to delay approval